### PR TITLE
Add Travis syntax check

### PR DESCRIPTION
**Pull request by sbesson**
This PR should simply enable Travis on the repository with a minimal `flake8` check

### CODE REVIEW

File: virtualmicroscope/urls.py [hunk 1]
```diff
@@ -24,9 +24,5 @@
 #
 
 from django.conf.urls import patterns
-from virtualmicroscope import views
```
Comment (aleksandra-tarkowska) [inline virtualmicroscope/urls.py]: You cannot remove this line as this is the way to load overridden view

**Comment by aleksandra-tarkowska** [comment]
Thanks @sbesson for taking care of the repo. Just small comment above.

File: virtualmicroscope/urls.py (hunk 1 above)
Comment (sbesson) [inline virtualmicroscope/urls.py]: Interesting, so `views` is imported but not used anywhere?

Comment (aleksandra-tarkowska) [inline virtualmicroscope/urls.py]: it is used, it overrides webstart/views/custom_index

Comment (sbesson) [inline virtualmicroscope/urls.py]: I meant not used directly in this file which is why it was flagged by `flake8`. Will look into that.

Comment (aleksandra-tarkowska) [inline virtualmicroscope/urls.py]: I presume ``  # NOQA `` would help flake8

Comment (sbesson) [inline virtualmicroscope/urls.py]: Can't think of a better view in the next 5 min. Pushed.

**Comment by aleksandra-tarkowska** [comment]
thank you, works as expected 
deployed with https://ci.openmicroscopy.org/job/OMERO-5.1-latest-vmicroscope/61/console

**Pull request merged**